src/csc439teamllama/cardgame/HoleEndPhase.java

```diff
@@ -0,0 +1,28 @@
+package csc439teamllama.cardgame;
+
+import java.util.Collections;
+
+/**
+ * This class collect all players cards,
+ * return them to the deck, shuffle the deck, and add new cards to players hands.
+ */
+public class HoleEndPhase implements Phase {
+    @Override
+    public void execute(GolfGameModel game, GolfView view ){
+         // game.scoreBoardUpdate();
```

Comment:
Whenever we merge all of our branches together we need to double check that all method names are correct. For instance, the method I created is called ScoreboardUpdate on my local system as opposed to scoreBoardUpdate. It's just something we will have to pay attention to when ultimately merging.